packages/core/jest.config.js

```diff
@@ -2,5 +2,5 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  testPathIgnorePatterns: ['/__fixtures__/', '/__utils__/'],
```

Comment:
why is `src` ignored

Reply:
because the `test.js` file is included by `jest` by default, while the `src` include test file may be  unexpected.

Reply:
Sure that `src` should not include any test files, but `ignore src` doesn't restrict the improper behavior.

Reply:
Any update on the `ignore specific test.ts file in src directory`?